/**:
  ros__parameters:
    lane_number: 3
    primitive_number: 5
    available_lanelet_id: [3590,  3570, 3513,
                          431,    7,    206,
                          975,    879,  927,
                          1316,   1268, 1271,
                          2689,   2688, 2620,
                          ]
    
    position:
      x: [244.3068,   241.2765,   238.47865,
          24.18835,   27.1351,    29.89505,
          -409.46725, -406.7692,  -404.32125,
          -804.32365, -803.01885, -801.5267,
          -560.24,    -562.89025, -565.36225,
        ]
        
      y: [-5.2726,    -7.625,     -9.9081,
          205.90695,  208.364,    210.7504,
          685.1625,   687.8067,   690.5283,
          873.10025,  876.74085,  879.98635,
          883.9446,   881.22515,  878.575,
        ]
      z: 0.0

    orientaion:
      x: 0.0
      y: 0.0
      z: [-0.7, 1.0, 1.0, 0.45, 0.7, 0.0] # todo 
      w: [0.7, 0.0, 0.0, 0.9, 0.7, 1.0] # todo

    desired_lane: [2, 2, 2, 2, 2]

    primitive_0: [3590,	3570,	3513,
                  3610,	3609,	3599,
                  2193,	2244,	2327,
                  2336,	2337,	2338,
                  2349,	2348,	2347,
                  431,	7,	  206,
                ]

    primitive_1: [431,	7,	  206,
                  463,	159,	277,
                  571,	395,	516,
                  3094,	3081,	3065,
                  3162,	3145,	3126,
                  3224,	3210,	3209,
                  3325,	3270,	3253,
                  975,	879,	927,
                ]

    primitive_2: [975,	879,	927,
                  1187,	1146,	1147,
                  1316,	1268,	1271,
                ] 

    primitive_3: [1316,	1268,	1271,
                  2492,	2484,	2382,
                  2502,	2497,	2419,
                  2689,	2688,	2620,
                ]

    primitive_4: [2689,	2688,	2620,
                  2971,	2965,	2911,
                  2989,	2990,	2944,
                  3011,	3010,	3009,
                  1634,	1624,	1625,
                  1869,	1804,	1746,
                  3361,	3351,	3350,
                  3402,	3392,	3383,
                  3455,	3439,	3426,
                  3590,	3570,	3513,
                ] 

    joker:
      position:
        x: -357.75685
        y: -5.52105
        z: 10.0
      orientaion:
        x: 0.0
        y: 0.0
        z: 1.0
        w: 0.0
      available_lanelet_id: [55555, 55555, 55555, 55555]
      primitive: [22736,  22743,  22750,  22757,
                  -1,     -1,     -1,     30333,
                  -1,     -1,     -1,     30438,
                  -1,     -1,     -1,     6838,  
                  -1,     -1,     -1,     32107, 
                  -1,     -1,     -1,     31992,  
                  11587,  11596,  11605,  11614,
                  33887,  33894,  33901,  33908,
                  11679,  11692,  11705,  11718,
                ]